sandboxes/react-rsbuild/src/stories/Bouton.stories.ts

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { fn } from '@storybook/test'
+import Bouton from './Bouton'
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+const meta = {
+  title: 'Example/Bouton',
```

Comment:
I think better to rename the new added story's title and file name to `ReExportButton` to help demonstrating its purpose.